src/main.ts

```diff
@@ -35,13 +34,14 @@ async function getChangeSet(client: AxiosInstance, workspaceId: string) {
   const changeSetsUrl = `/api/public/v0/workspaces/${workspaceId}/change-sets`
 
   let changeSetId = core.getInput('changeSetId')
-  let createdChangeSet = false
-  if (changeSetId === 'create') {
-    core.startGroup('Creating change set ...')
+  if (!changeSetId) {
+    core.startGroup('Creating new change set ...')
     const changeSetName = core.getInput('changeSetName')
```

Comment:
If we change the docs from `changeSetName` to `changeSet` we have to change this (and the example) too :)

Reply:
good catch!